test/config.json

```diff
@@ -104,7 +104,7 @@
 		},
 		"backup": {
 			"File": "unused.go",
-			"Args": ["vitess.io/vitess/go/test/endtoend/backup/vtctlbackup"],
+			"Args": ["vitess.io/vitess/go/test/endtoend/backup/vtctlbackup", "-timeout", "20m"],
```

Comment:
Do we still need to increase the timeouts?

Reply:
In some places I do but I increased in multiple of 10s, instead I can be more precise. Let me revise the timeout. Also now that we move test to stream what will help decrease the time-out for xtrabackup.